atest/run.py

```diff
@@ -119,9 +123,13 @@ def dos_to_long(path):
 
 
 if __name__ == '__main__':
+    coverage = False
+    if '--coverage' in sys.argv:
+        sys.argv.remove('--coverage')
+        coverage = True
```

Comment:
It probably doesn't matter, but modifying global `sys.argv` directly feels a bit bad. Perhaps this could be changed to something like this:

```python
args = sys.argv[1:]
if not args or '--help' in args:
    print(__doc__)
    rc = 251
elif '--coverage' in args:
    args.remove('--coverage')
    rc = atests(*args, coverage=True)
else:
    rc = atests(*args)
sys.exit(rc)
```